core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,32 @@
+const acceptanceTests = require("./utility-functions/puppeteer_utils.js");
+const testConstants = require("./utility-functions/testConstants.js");
+
+
+const blogEditOptions = "e2e-test-blog-post-edit-box";
+
+async function deleteDraftAsBlogAdmin() {
+  const user = await new acceptanceTests();
+  const page = await user.init();
+
+  await user.goto(testConstants.URLs.home);
```

Comment:
I think these five lines can go into a common utility function that's something like blogPostAdmin.signInWithEmail("testadmin@example.com"), since they'll be used in other tests.

Reply:
yes this thing will be common in all the tests. It can be in our parent utility class.. thanks!